security/i2pd/Portfile

```diff
@@ -51,10 +61,6 @@ platform darwin {
     }
 }
 
-# https://github.com/PurpleI2P/i2pd/issues/1798
```

Comment:
Maybe keep the issue ref though?

Reply:
The issue is outdated because now more versions of Clang need to be blocked due to another error.

Reply:
Ok, w/e. (For reviewers – this is not a blocker to merge.)